### DISCUSSION TITLE
how to enable search tab in vue press v2.0.0 beta 3.9?

### DISCUSSION THREAD

**Discussion by sachinshilwal** [vuepress/core#835] [Q&A] [answered]
So I am new to Vuepress and I am trying so stuff, and I was wondering how to install plugin for search in vuepress site in the navbar.
thank you.

**Comment by Mister-Hope** [comment, marked as answer]
https://v2.vuepress.vuejs.org/reference/plugin/search.html#install

**Reply by sachinshilwal** [reply]
now it is giving me this error
```
$ npm i -D @vuepress/plugin-search@next
npm ERR! code ETARGET
npm ERR! notarget No matching version found for ansi-styles@^4.3.4.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\LENOVO\AppData\Local\npm-cache\_logs\2022-04-17T15_37_44_059Z-debug.log
```

**Reply by Mister-Hope** [reply]
Check if you are using the npm registry. Or you may try to clean your cache. I am not reponsable for your own issues and I am not even interested in. Just claiming clearly, the issue you gives is not caused by our code.


https://stackblitz.com/edit/vuepress2-plugin-search?file=package.json

**Reply by sachinshilwal** [reply]
thanks man. It really helped me a lot. Solved
